### ISSUE TITLE
Consider renaming x/y/z to plane/row/col in the 3D structure tensor example

### ISSUE THREAD

**Issue by jni** [scikit-image/scikit-image#5275]
## Description

I didn't find any discussion in #5012 about axis naming, but to me it seems a bit muddled... We normally would call the vertical axis Y and the horizonal axis X, but in [the 3D structure tensor they are inverted](https://scikit-image.org/docs/dev/auto_examples/applications/plot_3d_structure_tensor.html).

Since we are striving to use planes/rows/columns throughout the library, I'm wondering @mkcor @grlee77 @rfezzani whether you would support changing the wording in that example to use that nomenclature. Happy to provide the PR for this if there is consensus.

**Comment by rfezzani** [comment]
The wording change is fine by me :wink:.

**Comment by mkcor** [comment]
@jni in that example, I intended the x/y 'inversion,' as made explicit with code line
```py
labels={'x': 'Y', 'y': 'X', 'color': 'intensity'}
```
but I'm definitely fine with replacing z/y/x by plane/row/col.

I 'inherited' the dataset from #4946 where I had derived that
```rst
Dimensions are provided in the following order: (z, y, x, c).
```
```py
n_plane, n_Y, n_X, n_chan = data.shape
```
see rendered (not yet merged) tutorial [here](https://1476-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/applications/plot_3d_interaction.html#sphx-glr-auto-examples-applications-plot-3d-interaction-py). Right now I can't remember what convinced me of the above -- @GenevieveBuckley would you have a clue by any chance? I agree it's confusing, because `(z, y, x)` would not be a directly oriented orthogonal triad...

**Comment by GenevieveBuckley** [comment]
I don't know, sorry @mkcor

**Comment by mkcor** [comment]
@GenevieveBuckley no worries! @jni I don't remember what crossed my mind and got me into a self-convincing loop... Anyhow, sorry for the confusion, I fixed it with #5282 (and 9656e41be5230a26efd67bfeff6adc1a98926296). I agree it's important to keep the plane/row/col[/channel] convention, consistently with the [tutorial introducing 3D image processing](https://scikit-image.org/docs/dev/auto_examples/applications/plot_3d_image_processing.html).

@GenevieveBuckley would you like me to use the #5282 opportunity to remove the explicit mention of your authorship, as in your [code review](https://github.com/scikit-image/scikit-image/pull/4946/commits/d4b070d8f587579cff79b32edbd9d92b1ae7c462)? Thanks!

**Comment by GenevieveBuckley** [comment]
> @GenevieveBuckley would you like me to use the #5282 opportunity to remove the explicit mention of your authorship, as in your [code review](https://github.com/scikit-image/scikit-image/pull/4946/commits/d4b070d8f587579cff79b32edbd9d92b1ae7c462)? Thanks!

Sure, go ahead
